docs/cli.md

```diff
@@ -22,6 +22,7 @@ Options:
                          'target_dir' configuration field.
   --dry-run              Run the tool without creating, changing, or deleting
                          any files.
+  --profiling TEXT       Path for profiling output. Switches on profiling.  
```

Comment:
Will move it from CLI into configurtation. I'll open a new PR after accepting this one.